tests/meta/test_v4_objects_imported.py

```diff
@@ -0,0 +1,48 @@
+"""
+Ensure objects defined in gitlab.v4.objects are imported in
+`gitlab/v4/objects/__init__.py`
+
+"""
+import os
+import pprint
+from typing import Set
+
+RELATIVE_V4_OBJECTS_DIR = os.path.join("gitlab", "v4", "objects")
+V4_OBJECTS_DIR = os.path.join(os.getcwd(), RELATIVE_V4_OBJECTS_DIR)
+V4_OBJECTS_INIT = os.path.join(V4_OBJECTS_DIR, "__init__.py")
+
+
+def parse_import_line(line: str) -> str:
+    line = line.rstrip()
+    assert line.startswith("from .")
+    line = line[len("from .") :]
+    return line.split()[0]
+
+
+def test_verify_v4_objects_imported() -> None:
+    print(os.getcwd())
+    print(V4_OBJECTS_DIR)
+    object_files: Set[str] = set()
+    for filename in sorted(os.listdir(V4_OBJECTS_DIR)):
+        if not filename.endswith(".py"):
+            continue
+        if filename in ("__init__.py",):
+            continue
+        root, _ = os.path.splitext(filename)
+        object_files.add(root)
```

Comment:
I think this part could also just be:
```python
import pkglib

import gitlab.v4.objects

modules =  [module.name for module in pkgutil.iter_modules(gitlab.v4.objects.__path__)]
```

Might need fewer constants and so on at the top then.

Reply:
Done. Mostly like that.